lib/new_relic/agent/transaction.rb

```diff
@@ -337,10 +337,8 @@ def set_default_transaction_name(name, category)
 
       def set_overriding_transaction_name(name, category)
         return log_frozen_name(name) if name_frozen?
-        if influences_transaction_name?(category)
```

Comment:
I think the removal of this check will probably work great. From what I can tell though, by removing this we will no longer have _any_ validation whatsoever on the user supplied category value. I'm thinking we should still perform some validation like so:

* Category is present
* Category is a symbol
* Category is found within an allow list
* Category is not on the "web" category list